src/services/membership/membership.dto.result.ecoverse.entry.ts

```diff
@@ -0,0 +1,27 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+import { MembershipResultEntry } from './membership.dto.result.entry';
+
+@ObjectType()
+export class MembershipEcoverseResultEntry {
```

Comment:
EcoverseMembershipPayload might be a better name for the given class in my opinion.

Reply:
Took this pattern from the search service - happy to have another name but lets land it quick.